### DISCUSSION TITLE
import path in .py

### DISCUSSION THREAD

**Discussion by abcitat** [sebastian-hofstaetter/teaching#141] [Advanced IR: Exercise Q&A]
So, i did part1 in google colab. I have a specific import paths to import files:
from google.colab import drive
drive.mount('/content/drive')
baseline = pd.read_csv('/content/drive/MyDrive/air-exercise-2/Part-1/fira-22.baseline-qrels.tsv........

I am not sure if i should just copy and paste my code in the in src/judgement_aggregation.py. Does the path matters in here (should i change anything?) or just the code matter?

**Comment by ldosen00** [comment]
Feel free to submit part one as a jupyter notebook. If you are then doing part two locally then you should alter the path in a way that you can load the data set properly because you will not be able to load it from google drive.
